Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/entity/ContactDiaryPersonEntity.kt

```diff
@@ -19,10 +20,13 @@ data class ContactDiaryPersonEntity(
         get() = personId
 }
 
+private const val MAX_CHARACTERS = 250
+private fun String.trimMaxCharacters(): String = this.trimToLength(MAX_CHARACTERS)
```

Comment:
This does not automatically remove whitespaces at start or end, and we want that right?
So `this.trim().trimToLength(....)`

Reply:
Ralf made a comment earlier that we shouldn't doo that: https://github.com/corona-warn-app/cwa-app-android/pull/2476#discussion_r583497360